deepmd/model/multi.py

```diff
@@ -0,0 +1,352 @@
+import numpy as np
+from typing import Tuple, List
+
+from deepmd.env import tf
+from deepmd.utils.pair_tab import PairTab
+from deepmd.utils.graph import load_graph_def
+from deepmd.env import global_cvt_2_ener_float, MODEL_VERSION, GLOBAL_TF_FLOAT_PRECISION
+from deepmd.env import op_module
+from .model import Model
+from .model_stat import make_stat_input, merge_sys_stat
+
+
+class MultiModel(Model):
```

Comment:
It is nice to introduce the `MultiModel`. Is it possible to write a function that can either a dict of model, as that the current `MultiModel` does, or a single model.  

I find the model building code in `MultiModel` is a duplication of the model making code in modules `model.**`

Reply:
I'll improve this in next PR.